### ISSUE TITLE
Spiralize and Brim with shell thickness

### ISSUE THREAD

**Issue by JohanBraeken** [daid/LegacyCura#847]
Hi,

This might be a bug, but it might also be possible I misunderstand the spiralize feature.

As I understand it, spiralize will create thin shells (and gradual Z increases). The "normal" shell thickness is most of the time the nozzle size, but it doesn't have to be.

According to the forum, when specifiing a shell thickness in Cura that is greater than the nozzle size, there is still only one pass made to print the object, but overextrusion is used to create thicker walls.

While printing an object with the spiralize feature and a shell size of 0.8 (2x nozzle size) I noticed a possible bug:

As the object was narrow, I used a brim.
It looks like the brim was printed with passes 0.8mm apart, but with no overextrusion.
Basically my brim was "lines" of 0.4mm width, with 0.4mm empty space in between them.
This didn't make a very stable/solid brim so I canceled the print. :-)

I think this is a bug, but I don't know if it is best to also use overextrusion for the brim, or to print the brim normally.

Cura 14.03 on Windows 8.1 64-bit.

Cheers,

**Comment by daid** [comment]
Most likely you are seeing problems because your feeder cannot supply enough material. When I review the GCode with spiralize+brim I see proper widths.
